From d93e7c3d74031fb2850ba9ae4e54c85776ecd013 Mon Sep 17 00:00:00 2001
From: Mike Looijmans <milo@openpli.org>
Date: Sat, 22 Jul 2017 19:35:00 +0200
Subject: [PATCH] setup.py: Remove /usr/local references for cross-compiling

Hard-coded /usr/... paths are bad when cross-compiling, so remove
them.

Signed-off-by: Mike Looijmans <milo@openpli.org>
---
 setup.py | 12 ------------
 1 file changed, 12 deletions(-)

diff --git a/setup.py b/setup.py
index 55c6847..f1e2a6f 100644
--- a/setup.py
+++ b/setup.py
@@ -33,18 +33,6 @@ home = os.path.expanduser("~")
 extens = [Extension('backports/lzma/_lzma',
                     ['backports/lzma/_lzmamodule.c'],
                     libraries = ['lzma'],
-                    include_dirs = [
-                        os.path.join(prefix, 'include'),
-                        os.path.join(home, 'include'),
-                        '/opt/local/include',
-                        '/usr/local/include'
-                    ],
-                    library_dirs = [
-                        os.path.join(prefix, 'lib'),
-                        os.path.join(home, 'lib'),
-                        '/opt/local/lib',
-                        '/usr/local/lib'
-                    ]
                     )]
 
 descr = "Backport of Python 3.3's 'lzma' module for XZ/LZMA compressed files."
-- 
2.7.4

